Makefile

```
@@ -191,6 +191,10 @@ batcher_start: ./batcher/aligned-batcher/.env user_fund_payment_service
 	@echo "Starting Batcher..."
 	@cargo +nightly-2024-04-17 run --manifest-path ./batcher/aligned-batcher/Cargo.toml --release -- --config ./config-files/config-batcher.yaml --env-file ./batcher/aligned-batcher/.env
 
+batcher_start_local: ./batcher/aligned-batcher/.env user_fund_payment_service
+	@echo "Starting Batcher..."
+	@cargo +nightly-2024-04-17 run --manifest-path ./batcher/aligned-batcher/Cargo.toml --release -- --config ./config-files/config-batcher.yaml --env-file ./batcher/aligned-batcher/.env.local
```

We could also make batcher_start_local run the local s3 as well:

```suggestion
batcher_start_local: ./batcher/aligned-batcher/.env user_fund_payment_service
        @echo "Starting local_db"
        @$(MAKE) run_storage &
	@echo "Starting Batcher..."
	@cargo +nightly-2024-04-17 run --manifest-path ./batcher/aligned-batcher/Cargo.toml --release -- --config ./config-files/config-batcher.yaml --env-file ./batcher/aligned-batcher/.env.local
```